storagetransfer/api/StorageTransfer.Samples.Tests/StorageFixture.cs

```diff
@@ -28,19 +32,84 @@ public class StorageFixture : IDisposable, ICollectionFixture<StorageFixture>
         public string ProjectId { get; }
         public string BucketNameSource { get; } = Guid.NewGuid().ToString();
         public string BucketNameSink { get; } = Guid.NewGuid().ToString();
+        public string JobName { get; }
+        public string SourceAgentPoolName { get; }
+        public string SinkAgentPoolName { get; }
+        public string GcsSourcePath { get; }
+        public string RootDirectory { get; } = System.IO.Path.GetTempPath();
+        public string DestinationDirectory { get; } = System.IO.Path.GetTempPath();
+        public string TempDirectory { get; } = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+        public string TempDestinationDirectory { get; } = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
         public StorageClient Storage { get; } = StorageClient.Create();
+        public string ManifestObjectName { get; } = "manifest.csv";
+        public string TopicId { get; } = "DotNetTopic" + Guid.NewGuid().ToString();
+        public string SubscriptionId { get; } = "DotNetSubscription" + Guid.NewGuid().ToString();
+        public string PubSubId { get; }
         public StorageTransferServiceClient Sts { get; } = StorageTransferServiceClient.Create();
 
+        public SubscriberServiceApiClient SubscriberClient { get; } = SubscriberServiceApiClient.Create();
+
+        public PublisherServiceApiClient PublisherClient { get; } = PublisherServiceApiClient.Create();
+
```

Comment:
How many of these are being used by more than one test? In the fixture, we only should add resources that are used by more than one test. Otherwise, the test itself should create and clean up the resources.